stdlib/Random/src/Random.jl

```diff
@@ -173,6 +202,15 @@ struct SamplerSimple{T,S,E} <: Sampler{E}
     data::S
 end
 
+"""
+    SamplerSimple(x, data)
+
+Create a sampler that wraps the given value `x` and the `data`, with the type information
+from `Random.gentype(x)` (which should be defined). This is the default fall-back for
+values.
```

Comment:
move "This is the default fall-back for values." to the `SamplerTrivial` docstring.